ga4gh/server/__init__.py

```diff
@@ -1,11 +1,14 @@
 from flask.ext.api import FlaskAPI
 from flask.ext.cors import CORS
+import os
 
 app = FlaskAPI(__name__)
-CORS(app, allow_headers='Content-Type')
 
-# TODO: Replace hard-coded value with a flag or conifg setting.
-app.config['MAX_CONTENT_LENGTH'] = 2 * 1024 * 1024  # 2MB
+app.config.from_object('ga4gh.server.config:DefaultConfig')
+if os.environ.get('GA4GH_CONFIGURATION') != None:
```

Comment:
Nit: we should use `is not None` instead of `!= None`. See https://www.python.org/dev/peps/pep-0008/#programming-recommendations